src/polyswarmd/config.py

```diff
@@ -1,4 +1,4 @@
-import json
+import ujson as json
```

Comment:
We definitely do not want to load, @sbnetops was right to point this out. But I think a better thing to do would be to explicitly use `ujson` instead of importing it as `json` so it's clear which is in use.